packages/terra-clinical-app-delegate/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "terra-clinical-app-delegate",
+  "main": "lib/app-delegate/AppDelegate.js",
+  "private": true,
+  "version": "0.0.0",
+  "description": "terra-clinical-app-delegate",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-clinical.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-clinical-app-delegate",
+    "AppDelegate",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-clinical/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-clinical#readme",
+  "devDependencies": {
+    "terra-toolkit": "^0.x"
+  },
+  "peerDependencies": {
+    "react": "^15.4.2"
+  },
+  "scripts": {
+    "compile": "npm run compile:clean && npm run compile:build",
+    "compile:clean": "$(cd ..; npm bin)/rimraf lib",
+    "compile:build": "$(cd ..; npm bin)/babel src --out-dir lib --copy-files",
+    "lint": "npm run lint:js",
+    "lint:js": "$(cd ..; npm bin)/eslint --ext .js,.jsx . --ignore-path ../../.eslintignore",
+    "release:major": "npm test && node ../../scripts/release/release.js major",
```

Comment:
[PR](https://github.com/cerner/terra-clinical/pull/49) for Lerna publish. These release scripts will need removed.